src/DataCollector/HttpDataCollector.php

```diff
@@ -19,12 +19,19 @@ class HttpDataCollector extends DataCollector
     /** @var \EightPoints\Bundle\GuzzleBundle\Log\LoggerInterface */
     protected $logger;
 
+    /**
+     * @var float
+     */
+    private $slowResponseTime;
+
     /**
      * @param \EightPoints\Bundle\GuzzleBundle\Log\LoggerInterface $logger
+     * @param float|null $slowResponseTime
      */
-    public function __construct(LoggerInterface $logger)
+    public function __construct(LoggerInterface $logger, $slowResponseTime = null)
     {
         $this->logger = $logger;
+        $this->slowResponseTime = $slowResponseTime ?: 0.5;
```

Comment:
We have default value in `src/DependencyInjection/Configuration.php`. Why do we need it here too and why `$slowResponseTime` is optional parameter here?

Reply:
yep, I thought about BC but also we have here the duplication of default value. OK, then probably we should keep it in mind and to drop in v8.

Reply:
it should be marked in a comment: "# todo remove in v8, PR #228"

Reply:
I guess here 0.5 sec is default value because in configuration 0.5 is default value too. To keep the same default value everywhere. In v8 we will remove default value here and only in config will remain. I guess... :slightly_smiling_face:

Reply:
In config it is 0 cause it's deactivated by default now. It should be 0 in HttpDataCollector.php also?!

Reply:
Yes, I'll change that in a few minutes